internal/server/edit_document.go

```diff
@@ -66,6 +68,11 @@ func EditDocument(client EditDocumentClient, tmpl template.Template) Handler {
 				if err != nil {
 					return err
 				}
+
+				sort.Slice(documents, func(i, j int) bool {
+					return documents[i].ID > documents[j].ID
+				})
```

Comment:
Would it be worth moving this sorting logic inside the `Documents(...)` function so theyre always sorted? Or is there ever a situation where we wouldn't want them sorted?